static/js/components/StarList.jsx

```diff
@@ -93,9 +93,18 @@ export const ObservingRunStarList = () => {
           )
         )
       );
-
+      const standard_promise = [
+        dispatch(
+          GET(
+            `/api/internal/standards?facility=${facility}`,
+            "skyportal/FETCH_STANDARDS"
+          )
+        ),
+      ];
       const starlistInfo = [];
       const values = await Promise.allSettled(promises);
+      const standard_value = await Promise.allSettled(standard_promise);
+      values.push(standard_value[0]);
```

Comment:
replace new stuff with 

```javascript
const standard_value = await dispatch(
          GET(
            `/api/internal/standards?facility=${facility}`,
            "skyportal/FETCH_STANDARDS"
          )
        );
```

Reply:
The suggested implementation hangs (starlist is never updated).

Reply:
Can you please try that again?  Danny's suggestion looks correct to me.